freqtrade/strategy/interface.py

```diff
@@ -304,6 +302,8 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                                               force_stoploss=force_stoploss, high=high)
 
         if stoplossflag.sell_flag:
+            logger.debug(f"'{trade.pair}' - Stoploss hit. Selling "
+                         f"(sell_type={stoplossflag.sell_type})...")
```

Comment:
Yes, I know of this 'non-lazy' behavior of f-strings and wanted to propose to change all of them in the backtesting/hyperopt to '%' format, which is lazy. But since they are already used -- I only introduced a few extra messages, for completeness and readability of the logs. This should not hit performance much. The transition to the '%' logging should be definitely done in a sep. PR.